code/modules/urist/modules/wfc/modules/deepmaint/event.dm

```diff
@@ -0,0 +1,109 @@
+/datum/event/backrooms_deepmaint
+	announceWhen = 10
+	endWhen = 1000
+	var/entrypoints = 100
+	var/list/spawned_teleporters
+
+
+/datum/event/gravity/setup()
+	endWhen = rand(600, 6000)
```

Comment:
Seems odd to overload the gravity event here. Error when copying an existing event as a template?
```suggestion
/datum/event/backrooms_deepmaint/setup()
	endWhen = rand(600, 6000)
```

Reply:
Oh, lol, yeah